client/gulpfile.js

```diff
@@ -57,6 +61,26 @@ const sources = {
   }
 };
 
+gulp.task('scss:compileJsToScss', () => {
+  const variablesConfigPath = 'scss/utilities/variables_config';
+  const compiledVariablesPath = 'scss/utilities/compiled_variables';
+  const files = fs.readdirSync(variablesConfigPath);
+
+  if (!fs.existsSync(compiledVariablesPath)) {
```

Comment:
We use `mkdirp` on the server.
https://github.com/wellcometrust/wellcomecollection.org/blob/cdc0c8e4e76aaf75a79a37dcbb82f3b40fd6ca0c/server/optimise-svgs.js#L20